CHANGELOG-DCA.rst

```diff
@@ -2,6 +2,44 @@
 Release Notes
 =============
 
+.. _Release Notes_7.53.0:
+
+7.53.0 / 6.53.0
+======
```

Comment:
Note: this seems to break formatting of titles (this was fixed in `main` but not on the 7.53.x branch, seems like the changelog generation still generates wrongly-formatted titles)